src/pages/index.js

```diff
@@ -44,39 +47,134 @@ const GalleryItem = styled.div`
 `;
 
 const IndexPage = () => {
+  const [new_magazine, setNewMagazine] = useState([]);
+  const [thumbnail_image, setThumbnailImage] = useState([]);
+
+  const months = [
+    "January",
+    "February",
+    "March",
+    "April",
+    "May",
+    "June",
+    "July",
+    "August",
+    "September",
+    "October",
+    "November",
+    "December",
+  ];
+  const domain = "https://sparkchive.vercel.app/";
+
+  useEffect(() => {
+    const sparkMagazine = async () => {
+      await fetch(
+        "https://api.github.com/repos/prince-deriv/deriv-static/contents/public/magazine"
```

Comment:
urls can be assigned as constants on top so it will be easier for us to update it later, something like 

```
const MAGAZINE_URL = ''

Reply:
okay